templates/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -642,6 +649,9 @@ export class {{ service.name }}Client {
     ) as AsyncIterable<{{ util.toInterface(method.pagingResponseType) }}>;
   }
 {%- endfor %}
+{%- if service.iamService %}
+{{ util.iamServiceMethods()}}
```

Comment:
nit: `util` is really overloaded with all the kinds of random stuff. Maybe make `_iam.njk` (starting with underscore so it gets ignored by the templater) and put both service methods and tests there?

Reply:
got it, it makes perfect sense to move them to another njk file. Thank you for the suggestion!

Reply:
done.